src/cli/commands/force/lightning/local/device/create.ts

```diff
@@ -165,6 +163,20 @@ export class Create extends Setup {
             });
     }
 
+    private _requirements: CommandRequirements = {};
+    public get commandRequirements(): CommandRequirements {
+        if (Object.keys(this._requirements).length === 0) {
+            const commandDict: CommandRequirements = {};
+            commandDict.setup = getPlatformSetupRequirements(
+                this.logger,
+                this.flags.platform,
+                this.flags.apilevel
+            );
+            this._requirements = commandDict;
+        }
+        return this._requirements;
+    }
```

Comment:
For now each command has its own private `_requirements` as the base class `Setup`'s property is private and cannot be accessed.